docs/management/maintenance-windows/maintenance-windows.asciidoc

```diff
@@ -46,4 +46,6 @@ A maintenance window can have any one of the following statuses:
 - `Upcoming`: It will run at the scheduled date and time.
 - `Running`: It is running.
 - `Finished`: It ended and does not have a repeat schedule.
-- `Archived`: It is archived. In a future release, archived maintenance windows will be queued for deletion.
\ No newline at end of file
+- `Archived`: It is archived. In a future release, archived maintenance windows will be queued for deletion.
+
+When you <<rule-details,view alert details>> in {kib}, each alert shows a unique identifier for the most recent window that affected it.
```

Comment:
Actually, we will show all of the maintenance window IDs that affect any particular alert, as the `maintenance_window_ids` field is an array type.

Reply:
Thanks for the clarification! I've updated the text via d3add68657c7649efb23d721654d516d220baca9